Leanplum-SDK/Classes/Managers/Networking/LPNetworkEngine.m

```diff
@@ -109,9 +109,6 @@ - (void)setHostName:(NSString *)hostName
                                     cachePolicy:NSURLRequestReloadIgnoringLocalCacheData
                                     timeoutInterval:timeout];
     request.HTTPMethod = method;
-    [request setValue:[NSString stringWithFormat:@"%@, en-us",
```

Comment:
any reason to move this to request?

Reply:
to be consistent with the way sessionconfiguration is being used to set http headers.

Reply:
It was already setting @"User-Agent": userAgentString , there. So, i added rest of the headers in there for both requests.